README.rst

```diff
@@ -80,15 +81,16 @@ The following is the output from ``ansible-lint --help``, providing an overview
                             to specify this if no -r flags are used
       -t TAGS               only check rules whose id/tags match these values
       -T                    list all the tags
+      -v                    Increase verbosity level
       -x SKIP_LIST          only check rules whose id/tags do not match these
                             values
+      --nocolor             disable colored output
+      --force-color         Try force colored output (relying on ansible's code)
       --exclude=EXCLUDE_PATHS
                             path to directories or files to skip. This option is
                             repeatable.
-      --force-color         Try force colored output (relying on ansible's code)
-      --nocolor             disable colored output
-      -c /path/to/file      Specify configuration file to use.  Defaults to
-                              ".ansible-lint"
+      -c C                  Specify configuration file to use.  Defaults to
```

Comment:
maybe `CONFIGFILE` or existing `/path/to/file` is a bit more user-friendly

Reply:
Yeah, I don't know what C is. That was the literal output on my macbook.

Reply:
I will also make this more readable in `--help`